build/tests/service-test.sh

```diff
@@ -0,0 +1,228 @@
+#!/bin/bash
+
+# Copyright 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# -----------------------------------------------------------------
+# -----------------------------------------------------------------
+source ${PDO_SOURCE_ROOT}/bin/lib/common.sh
+check_python_version
+
+# -----------------------------------------------------------------
+# -----------------------------------------------------------------
+if [ "${PDO_INTERPRETER}" == "wawaka-aot" ]; then
+    die Automated tests for the wawaka-aot interpreter are currently not supported.
+fi
```

Comment:
This should be removed.

Reply:
that's your issue. i'm just copying code.

Reply:
done.